### PR TITLE
Spec for Matrix#to_a

### PR DESCRIPTION
**Pull request by rashmirathi1729**

### CODE REVIEW

**Comment by eregon** [comment]
`it_behaves_like` does not take a block, please look at existing code (hint: grep for `matrix_identity`)

**Comment by rashmirathi1729** [comment]
@eregon Ah! it's already tested. Should I just make a pull request for Matrix#to_a spec?

**Comment by eregon** [comment]
yes :smiley:

**Comment by eregon** [comment]
(or remove the other commit)

**Comment by rashmirathi1729** [comment]
@eregon done

File: library/matrix/to_a_spec.rb [hunk 1]
```diff
@@ -2,5 +2,7 @@
 require 'matrix'
 
 describe "Matrix#to_a" do
-  it "needs to be reviewed for spec completeness"
+  it "returns the array of arrays that describe row of the matrix" do
```
Comment (eregon) [inline library/matrix/to_a_spec.rb]: `that describe *the rows* of the matrix` reads better I think

File: library/matrix/to_a_spec.rb [hunk 2]
```diff
@@ -2,5 +2,7 @@
...
+    Matrix[[1, 0], [0, 1]].to_a.should == [[1, 0],[0, 1]]
```
Comment (eregon) [inline library/matrix/to_a_spec.rb]: Could use different values for each cell to strengthen the spec?

**Comment by eregon** [comment]
I think it would also be interesting to add examples with empty and single-element matrices.

**Comment by rashmirathi1729** [comment]
@eregon What do you think now?

**Comment by eregon** [comment]
Looks great, thanks for your contribution! :+1:

**Pull request merged**